dsl/views.go

```diff
@@ -1838,6 +1840,36 @@ func Position(pos int) {
 	}
 }
 
+// Implementation sets the implementation of the automatic layout, defaults to ImplementationDagre.
+//
+// Implementation must appear in AutoLayout.
+//
+// Implementation takes one argument: the ImplementationKind.
+//
+// Example:
+//
+//	var _ = Design(func() {
+//	    var System = SoftwareSystem("Software System", "My software system.")
+//	    Views(func() {
+//	        SystemContextView(SoftwareSystem, "context", "An overview diagram.", func() {
+//	            AutoLayout(func() {
+//	                Implementation(ImplementationGraphviz)
+//	            })
+//	        })
+//	    })
+//	})
+func Implementation(impl ImplementationKind) {
+	if reflect.ValueOf(impl).IsNil() {
```

Comment:
Not sure I understand why this test is needed? Isn't ImplementationKind a int? I believe this check will cause a panic.

Reply:
Very valid, since you need to provide any ImplementationKind and can't just leave it empty. ➡️removed